coordinator/src/main/scala/filodb.coordinator/IngestionActor.scala

```diff
@@ -60,6 +60,7 @@ private[filodb] final class IngestionActor(dataset: Dataset,
 
   final val streamSubscriptions = new HashMap[Int, CancelableFuture[Unit]]
   final val streams = new HashMap[Int, IngestionStream]
+  private var shardStateVersion: Long = 0
```

Comment:
Just a note: in Actor classes everything is essentially private, because the direct reference to an instance of an Actor class is NEVER shared.  The only way to create an actor is via `actorOf`, which returns strictly an `ActorRef` -- thus Actor classes, methods, and fields are never directly accessible.  I'm fine with "private" to communicate intent I guess.

Reply:
I can switch it to be consistent. The default Scala behavior is for everything to be public by default. It would be nice it everything was private by default. Oh well.